src/style.scss

```diff
@@ -2,4 +2,7 @@
   margin: 0;
   padding: 0;
   background: #000;
-}
\ No newline at end of file
+}
+canvas{
+  border: 2px dashed orange;
+}
```

Comment:
C'est pour checker que l'image est bien centrée ou bien?

Reply:
Oui, à virer (j'avais testé des media queries basées sur l’aspect ratio du genre `@media screen and (max-device-aspect-ratio: 4/3)`